library/control/src/modules/WorkflowManager.rb

```diff
@@ -478,6 +467,46 @@ def addon_control_dir(src_id, cleanup: false)
       dir
     end
 
+    # Path of the control file contained in the package that has been previously
+    # extracted to the given directory
+    #
+    # @see #control_file
+    #
+    # @param dir [String] directory where the package has been extracted to
+    # @return [String] name of the control file
+    def control_file_at_dir(dir)
```

Comment:
It seems to me a bit weird (aka confusing) to have `control_file_at_dir` and `control_file_at` methods, both receiving the _dir_ parameter. 

What about changing the first to something similar to `find_control_file_for`... not a great name, but "slightly" different to the other. Thoughts?